save-frontend/src/main/kotlin/org/cqfn/save/frontend/utils/RequestUtils.kt

```diff
@@ -61,18 +63,21 @@ suspend fun post(url: String, headers: Headers, body: dynamic) = request(url, "P
  * @param method HTTP request method
  * @param headers HTTP headers
  * @param body request body
+ * @param credentials
  * @return [Response] instance
  */
 suspend fun request(url: String,
                     method: String,
                     headers: Headers,
                     body: dynamic = undefined,
+                    credentials: RequestCredentials? = undefined,
```

Comment:
why nullable if you have the default value?
or may be not provide default value here

Reply:
Because `undefined` is `Nothing?`, and it's the default value in `RequestInit`